### DISCUSSION TITLE
Godbolt text entry on iPhone Chrome not working

### DISCUSSION THREAD

**Discussion by jonnygrant** [compiler-explorer/compiler-explorer#5011] [Ideas]
Hello
Has anyone had success using godbolt.org via Chrome on iPhone? I wanted to try something out.

The keyboard doesn't pop-up, so couldn't paste in my code.

Also, would be great if it could set the windows vertically titled, so I could scroll down, and have a full page to on each window, like the code entry window  etc

Screenshot below

![godbolt__IMG_0124](https://user-images.githubusercontent.com/29703560/235350796-2d2a8a11-95b9-433f-94dd-2be8779dd922.jpg)

**Comment by jakirkham** [comment]
Would just like to add am seeing the same issue in Safari on iPhone

**Comment by mattgodbolt** [comment]
This is on purpose- the underlying text widget doesn't support mobile at all, and so we go into readyonly mode. If you really wnant to try you can "request desktop site" but it won't end well, sorry.

See #4571
